Cron/Feed.php

```diff
@@ -247,8 +250,13 @@ public function execute()
 
                                 $this->printLine($siteId, '<mpn><![CDATA['.$this->escapeString($childProduct->getSku()).']]></mpn>', 5);
 
-                                $stock_item = $this->_stockItem->get($childProduct->getId());
-                                $this->printLine($siteId, '<stock>'.($stock_item && $stock_item->getIsInStock() ? ($stock_item->getQty() > 0 ? (int) $stock_item->getQty() : 1) : 0).'</stock>', 5);
+                                try {
+                                    $stock_item = $this->_stockItem->get($childProduct->getId());
```

Comment:
Can you change this to set the stock level as a variable and only print the line once please?

Reply:
Yes